Neos.Flow/Classes/Mvc/ActionResponse.php

```diff
@@ -0,0 +1,125 @@
+<?php
+namespace Neos\Flow\Mvc;
+
+use Neos\Flow\Annotations as Flow;
+use Neos\Flow\Http\Helper\ResponseInformationHelper;
+use Neos\Flow\Http\Response;
+use Psr\Http\Message\UriInterface;
+
+/**
+ * The new minimal MVC response object.
+ * For anything more use a custom HTTP component and set a component parameter.
+ *
+ * @Flow\Proxy(false)
+ * @api
+ */
+final class ActionResponse extends \Neos\Flow\Http\Response implements ActionResponseInterface
```

Comment:
extends `Http\Response` gives me the shivers, even if it's just for now to be b/c 🙈